src/ai/backend/common/typed_validators.py

```diff
@@ -156,3 +158,26 @@ def session_name_validator(s: str) -> str:
 
 SessionName = Annotated[str, AfterValidator(session_name_validator)]
 """Validator with extended re.ASCII option to match session name string literal"""
+
+
+def _vfolder_name_validator(name: str) -> str:
+    """
+    Although the length constraint of the `vfolders.name` column is 128,
+    we limit the length to 64 in the create/rename API
+    because we append a timestamp of deletion to the name when VFolders are deleted.
+    """
+    if (name_len := len(name)) > 64:
```

Comment:
Rather than using hardcoded constant values, it would be better to declare and use them as Const values.
This improves the readability of the logic. Additionally, if the specifications change in the future, it will be easier to make consistent updates.